packages/launcher/package.json

```diff
@@ -61,14 +61,13 @@
     "mime": "^2.4.0",
     "qrcode.react": "^0.9.3",
     "react": "^16.8.5",
-    "react-addons-css-transition-group": "^15.6.2",
     "react-art": "^16.8.5",
     "react-blockies": "^1.4.0",
-    "react-copy-to-clipboard": "^5.0.1",
     "react-dom": "^16.8.5",
     "react-modal": "^3.6.1",
     "react-native-web": "^0.10.0",
     "react-relay": "^1.7.0",
+    "react-transition-group": "1.x",
```

Comment:
Looks like the latest version is v4, any reason to stick to v1 here please?

Reply:
well I just used the one suggested in the docs you sent me :)

Reply:
Ah sorry I didn't notice the link was to a specific branch, do you mind upgrading to the latest version so it's easier to maintain later please?